angular-ngrx-scss/src/app/repository/components/repo-header/repo-navigation/repo-navigation.component.scss

```diff
@@ -0,0 +1,68 @@
+@use '../../../../shared/styles/variables.scss';
```

Comment:
issue: Leaving this here since it's CSS related - I don't think our active tab highlight is working _quite_ like we'd expect. In the deploy preview, I'm seeing some odd alternation on hover, and when clicking on the issues route, the highlight stays under the code tab instead of toggling off. 

Not sure exactly what's causing it, but worth looking into! 

https://user-images.githubusercontent.com/16214572/184669590-6132e05b-c852-4ad6-aa28-98cc12f59bbe.mov

Reply:
solved.